modules/rules_spring/2.6.0/MODULE.bazel

```diff
@@ -0,0 +1,67 @@
+module(
+    name = "rules_spring",
+    version = "2.6.0",
+    compatibility_level = 2,
+    repo_name = "rules_spring",
+)
+
+# Python for dupe class checking support
```

Comment:
Not related to this PR, but you may be interested in `--experimental_one_version_enforcement`.

Reply:
Nice, yes for rules_spring dupe class checking is our key feature. It has caught so many conflicts before they became problems in production.